autoload/go/complete.vim

```diff
@@ -148,11 +137,6 @@ function! go#complete#GetInfoFromOffset(offset)
     return ""
 endfunction
 
-function! go#complete#GetInfo()
-    let offset = go#complete#gocodeCursor()+1
-    return go#complete#GetInfoFromOffset(offset)
-endfunction
```

Comment:
This is a public function that people use, for example to plugin into statusline or use for their own functions.